cc/testing/evaluation/WORKSPACE.bazel

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+""" Workspace for Open Source differential privacy library """
```

Comment:
I ended up deleting this WORKSPACE file, as the CC file did not compile successfully while it was in the folder (just pushed the change). I checked and the testing folder also does not have a WORKSPACE file. Instead both default to the WORKSPACE in the CC folder. So this may be a moot point.

Reply:
Unless we feel that the folder needs a WORKSPACE file? @dasmdasm?

Reply:
Nope, not having a workspace is fine.